libs/template/templates/default-python/template/{{.project_name}}/README.md.tmpl

```diff
@@ -0,0 +1,36 @@
+# {{.project_name}}
+
+The '{{.project_name}}' project was generated using the default-python template.
+
+## Getting Started
+
+1. Install the Databricks CLI from https://docs.databricks.com/en/dev-tools/cli/databricks-cli.html
+
+2. Authenticate to your Databricks workspace:
+    ```
+    $ databricks auth login
+    ```
```

Comment:
This command works only for OAuth, not to configure PATs.

We should look into consolidating the OAuth flow and the PAT configuration flow.

cc @mgyucht

Reply:
Ok, let's track that as an issue? See also DECO-2579 for a related ticket on customer-facing errors.

Reply:
Ok, we're tracking this, let me resolve this.

Reply:
Please update to `databricks configure` until we have a better entry point that includes both OAuth and PATs.

OAuth is not yet widely adopted and we know for sure everyone is able to use PATs.